### ISSUE TITLE
achilles ignoring fetchSize set in raw Statement 

### ISSUE THREAD

**Issue by fortm** [doanduyhai/Achilles#323]
WIth raw() , query need to set Fetch Size and Paging State, but it seems achilles is ignoring fetch Size in this case and returns all results. Unable to use dsl() in my case due to [bug](https://github.com/doanduyhai/Achilles/issues/320)

```
final SimpleStatement statement = new SimpleStatement("select * from xyz where xid=:xid  ");
statement.setPagingState(state);
statement.setFetchSize(fetchSize);
Tuple2<List<XYZ>, ExecutionInfo> page = eventByAgentManager
                    .raw()
                    .typedQueryForSelect(statement, xid)
                    .getListWithStats();
```

**Comment by doanduyhai** [comment]
Can you please give me some steps to reproduce locally ?

**Comment by fortm** [comment]
@doanduyhai , all I did was create table xyz with secondary index xid in cassandra
Then due to bug mentioned in ticket, I had to use raw query.

I needed pagination like dsl() with raw() api. So set fetchSize to 10
But page._1() has list of size 200 which is total number of records.
I was expecting it to return 10 in list item.

**Is there any other way we should set paging state and fetchSize when using raw() api  than below ?**

Here XYZ matches columns from xyz table and is POJO
> int fetchSize = 10;
> final SimpleStatement statement = new SimpleStatement("select * from xyz where xid=:xid  ");
> statement.setFetchSize(fetchSize);
> Tuple2<List<XYZ>, ExecutionInfo> page = eventByAgentManager
>                     .raw()
>                     .typedQueryForSelect(statement, xid)
>                     .getListWithStats();

**Comment by fortm** [comment]
@doanduyhai, is there any other way we can use raw() api as well as set pagingstate and fetchSize  than above ?

**Comment by doanduyhai** [comment]
Ok I see the problem is here: https://github.com/doanduyhai/Achilles/blob/master/achilles-core/src/main/java/info/archinnov/achilles/internals/runtime/AbstractManager.java#L178-L192

Indeed, from the simple statement, we prepare it and execute the prepared statement.

However we **did not** copy all the query options from the regular statement to the newly generated bound statement. This is indeed a **bug**

Fortunately there is  a simple work-around: inject a **`BoundStatement`** instead of the **`RegularStatement`**. This means that:

* you'll need to prepare and bind the statement yourself but it's not a big deal since **Achilles** manager always expose the underlying **`Session`** object via `manager.getNativeSession()`
* **Achilles** can no longer display bound values in the DDL query logger

**Comment by doanduyhai** [comment]
Fixed in Achilles 6.0.0
